src/main/java/org/jboss/forge/arquillian/container/provider/ContainerDirectoryParser.java

```diff
@@ -23,7 +25,7 @@
     private static List<Container> containers;
 
     @Inject
-    private ContainerIndexLocationProvider containerDirectoryLocationProvider;
+    ContainerIndexLocationProvider containerDirectoryLocationProvider;
 
     void createContainers() {
```

Comment:
Is it really what it does? Create containers? Or does it look it up? Also how about making it to return the collection of containers and expose as public method instead of keeping the state in static? Is it expensive operation?